certbot-apache/certbot_apache/_internal/parsernode_util.py

```diff
@@ -30,7 +30,7 @@ def validate_kwargs(kwargs: Dict[str, Any], required_names: List[str]) -> Dict[s
     return validated_kwargs
 
 
-def parsernode_kwargs(kwargs: Dict[str, Any]) -> Tuple[Any, ...]:
+def parsernode_kwargs(kwargs: Dict[str, Any]) -> Tuple[Any, Any, Any, Any]:
```

Comment:
As mentioned earlier, I feel that these would benefit from typing, if it's possible.

Reply:
Clearly you are right. I did that.